homeassistant/components/coinbase/const.py

```diff
@@ -3,6 +3,8 @@
 CONF_CURRENCIES = "account_balance_currencies"
 CONF_EXCHANGE_BASE = "exchange_base"
 CONF_EXCHANGE_RATES = "exchange_rate_currencies"
+CONF_EXCHANGE_PRECISION = "exchange_rate_precision"
+CONF_EXCHANGE_PRECISION_DEFAULT = 2
```

Comment:
We only prefix constants that represent config keys with `CONF_`. Default values we usually prefix with `DEFAULT_`.